### DISCUSSION TITLE
How to render a code block containing quarto markdown div code?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-cli#8508] [Q&A] [answered]
### Description

For example, what code should I write in a `qmd` file to render the following `code block` showing some `quarto markdown div`, including those `:::`, or \```{python}` marks, without actually evaluating it as it is? 

<img width="937" alt="Screenshot 2024-01-30 at 23 39 32" src="https://github.com/quarto-dev/quarto-cli/assets/23435810/3a6add1b-779b-4178-a270-446c669e26e9">

**Comment by cderv** [comment, marked as answer]
This is the part of the doc that shows the syntax to use unexecuted code block for demo 
https://quarto.org/docs/computations/execution-options.html#unexecuted-blocks

Also you seem to be referring to our doc website, source is available by the way
https://github.com/quarto-dev/quarto-web/blob/aa0e031d66beca0b772cb25640a5a81dab92528f/docs/output-formats/html-basics.qmd

**Reply by cscheid** [reply]
Also, in our website, all the pages generated from`.qmd` files have a link to "edit this page", which will take you to a github page where you can see the source code.

**Comment by hopezh** [comment]
Thank you.

I think the use of four backtick symbols is what I should learn from your suggestion. 

<img width="1262" alt="Screenshot 2024-01-31 at 00 33 27" src="https://github.com/quarto-dev/quarto-cli/assets/23435810/aadab6fd-6ded-4716-9c6b-df09876774e6">

<img width="1438" alt="Screenshot 2024-01-31 at 00 34 09" src="https://github.com/quarto-dev/quarto-cli/assets/23435810/0ba7fa14-a460-4b9e-9d49-77863826defd">

--- 

However, after I used it in a qmd file, there is a highlight color for the code block symbol ```, as show below on my quarto website test page: https://hopezh.github.io/oatNotes/quarto_tabset.html

<img width="1369" alt="Screenshot 2024-01-31 at 00 35 07" src="https://github.com/quarto-dev/quarto-cli/assets/23435810/c463e448-6f8f-441d-acf9-9250a866f498">

May I ask how to get rid of the highlight color for the 3-backtick lines? 

Thanks.

**Reply by mcanouil** [reply]
See <https://quarto.org/docs/output-formats/html-code.html#highlighting>.
The colour, keywords, etc. is not Quarto.
This is done by Pandoc via Skylighting.

**Reply by cderv** [reply]
To be more precise, you used 
`````
```` markdown
`````

which will apply the markdown highlighting. But this highlighting is not aware of special code chunk syntax bring by Quarto 
````
```{python}
````

So you need to change the language used here on this block. Set no language, or use `default` as mentioned in the doc: 
https://quarto.org/docs/authoring/markdown-basics.html#source-code

Maybe we'll manage to adapt the syntax highlighter to support this specific code chunk syntax. I'll check this
